dashboard/lib/tasks/seed.rake

```diff
@@ -154,6 +154,67 @@ namespace :seed do
     seed_regional_partners_school_districts(true)
   end
 
+  task create_pd_dashboard_data: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    # Destroy existing facilitator and organizers
+    User.where(email: ['pd_test_facilitator@code.org', 'pd_test_organizer@code.org']).destroy_all
+
+    # Make a facilitator
+    facilitator = create(:facilitator, email: 'pd_test_facilitator@code.org')
+    Pd::CourseFacilitator.create(facilitator_id: facilitator.id, course: Pd::Workshop::COURSE_CSP)
+
+    # Make a workshop organizer
+    organizer = create(:workshop_organizer, email: 'pd_test_organizer@code.org')
+
+    # Create an ended workshop
+    workshop = create(:pd_workshop, course: Pd::Workshop::COURSE_CSP, num_sessions: 1, enrolled_and_attending_users: 5, organizer: organizer)
+    workshop.facilitators << facilitator
+    workshop.start!
+    workshop.end!
+
+    # Fill in survey questions for this workshop
+    responses = {}
+
+    [
+      Api::V1::Pd::WorkshopScoreSummarizer::FACILITATOR_EFFECTIVENESS_QUESTIONS,
+      Api::V1::Pd::WorkshopScoreSummarizer::TEACHER_ENGAGEMENT_QUESTIONS,
+    ].flatten.each do |question|
+      responses[question] = PdWorkshopSurvey::OPTIONS[question].last
+    end
+
+    Api::V1::Pd::WorkshopScoreSummarizer::OVERALL_SUCCESS_QUESTIONS.each do |question|
+      responses[question] = PdWorkshopSurvey::AGREE_SCALE_OPTIONS.last
+    end
+
+    begin
+      workshop.enrollments.each do |enrollment|
+        PEGASUS_DB[:forms].insert(
```

Comment:
This makes dashboard seed depend on Pegasus.  Why does the data need to be there instead of stubbed in the tests?